### DISCUSSION TITLE
Order of fields displayed in relationship fields [Jun 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5888] [Discord Threads]
> 🙋‍♂️ **V.S. (He/Him)**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Not supported currently. Are you aware of any other vendor who does that? curious to know your use case.  
#  
> 🙋‍♂️ **V.S. (He/Him)**  
#### Haven’t tried other vendors yet. My use case is that I’m working for a small voluntary organisation and we are looking for a case management solution that we can adapt to our needs. Most case management or CRM platforms available are not fit for purpose as they either have a lot of marketing and analytics tools we don’t need, or they don’t capture the relevant data in a robust or effective way.   
####    
#### Generally NocoDB seems great for what we need. A few things could greatly improve the UX for this purpose. Like having a field populate automatically based on data from other fields (for generating an anonymised case number based on the time stamp for submission and case category) or reordering the fields within a relationship field modal (for ease of use when doing data entry). The former isn’t huge, but the lack of the latter makes data entry really annoying.  
#  
> 🙋‍♂️ **V.S. (He/Him)**  
#### So far I’m considering two workarounds. First is rebuilding the database with the tables such that the columns are defined in precisely the right order for the fields. Or secondly changing from an SQLite database in the testing environment to MySQL and using CLI tools to rearrange the columns and then re syncing metadata. Not sure if either of these would work. But it’s worth a shot.  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### > Like having a field populate automatically based on data from other fields   
#### Try Formula field.    
####    
#### For reordering fields within a relationship - there is a roadmap item to limit displayed entries in a relationship modal to a view & hence with that, ordering and list of items to be displayed can be easily controlled by associating link with a specific view. That should help - but not in our immediate priority list.  
#  
> 🙋‍♂️ **V.S. (He/Him)**  
#### Okay, so even when I rearrange the rows of the table and sync metadata, the order of the fields is completely messed up. So it appears that until this feature of the relationship modal being limited to a view is added in the roadmap I won't be able to use nocoDB for my use case. Which is unfortunate  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1104338478575141024)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
